crates/core/src/db/datastore/system_tables.rs

```diff
@@ -169,14 +167,14 @@ st_fields_enum!(
 st_fields_enum!(enum StConstraintFields {
     "constraint_id", ConstraintId = 0,
     "constraint_name", ConstraintName = 1,
-    "kind", Kind = 2,
+    "constraints", Constraints = 2,
     "table_id", TableId = 3,
     "columns", Columns = 4,
 });
 // WARNING: For a stable schema, don't change the field names and discriminants.
 st_fields_enum!(enum StModuleFields {
     "program_hash", ProgramHash = 0,
-    "kind", Kind = 1,
+    "constraints", Kind = 1,
```

Comment:
It's a bit odd that this one did not change both, but the equivalent in `StConstraintsFields` did. But maybe that's an artifact of the split from #267?